src/api/handlers/action.py

```diff
@@ -90,6 +90,19 @@ def submit(self, request):
       .expect("image", "file", is_required=False, options={"is_logo": True})
       .expect("vendors", list, is_required=False)
       .expect("action_id", str, is_required=False)
+      .expect(
```

Comment:
instead of all this code, can the front-end pass in 'media_info' as a json structure, which then goes into media_info?   No need to see guardian and copyright stuff as part of an action or a vendor or all the places we create content.

Reply:
Thats is possible, its just NOT good. We want to be able to validate all the parts.